playbook-website/.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-typescript"
+  ],
+  "plugins": ["react-docgen"]
+}
```

Comment:
`babel.config.js` was removed as it is no longer needed (Vite takes care of this), but this file is necessary for `react-docgen` to properly parse our kits.